mods/seasonal/random-teams.js

```diff
@@ -19,6 +19,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: 'Kamikaze Rebirth',
 				evs: {hp: 172, atk: 228, spe: 108}, nature: 'Adamant',
 			},
+			'panpawn': {
+				species: 'Cyndaquil', ability: 'Flash Fire', item: 'Leftovers', gender: 'M', //ask gender
+				moves: ['Eruption', 'Extrasensory', 'Facade'],
+				signatureMove: 'LaFireBlaze',
+				evs: {}, /* he said skip */ nature: 'Adamant',
```

Comment:
Id ask him, he probably forgot EVs, though if he really dosent want EVs just remove the property all together so the generator will default to 84 all.